mlflow/tensorflow.py

```diff
@@ -37,6 +37,7 @@
 from mlflow.utils.file_utils import _copy_file_or_tree
```

Comment:
In tensorflow autologging, `_manage_active_run` and `_AUTOLOG_RUN_ID` manage an active run across the patched functions.

https://github.com/mlflow/mlflow/blob/d1abdc2ee492958c7376db43c0be4780d403c380/mlflow/tensorflow.py#L854-L900

Reply:
@dbczumar Is it possible to patch multple functions with `safe_patch` and share the same active run across them?

Reply:
↑ Just never mind

Reply:
The current tensorfolw autologging works like this (maybe I'm wrong):

```python
# no active run exists

classifier.train(input_fn=lambda: input_fn(train, train_y, training=True), steps=training_steps)
# -> creates a new run, store the run_id in `_AUTOLOG_RUN_ID`, and terminates it

classifier.export_saved_model(directory, receiver_fn)
# -> Restart the run using `_AUTOLOG_RUN_ID` and save the trained model in it
```